packages/core/src/interfaces/identifiable.ts

```diff
@@ -0,0 +1,5 @@
+export type Identifier = { toString(): string };
```

Comment:
i feel like this should be named Id, it's Id everywhere, (i.e EventId and not EventIdentifier even though its an identifier)